### PR TITLE
Adds laughter demon to wizard's spellbook

### PR DESCRIPTION
**Pull request by coiax**
:cl: coiax
rscadd: Wizards have been experimenting with summoning the slightly less
lethal "laughter demon". Please be aware that these demons appear to be
nearly as dangerous as their slaughter cousins. But adorable, nonetheless.
/:cl:

By popular demand.
Cheaper than a slaughter demon because it's not going to kill the crew,
it might even end up healing a bunch of people.

### CODE REVIEW

File: tools/linux_build.py [hunk 1]
```diff
@@ -31,7 +31,7 @@ def stage2(map):
         txt = "-M{}".format(map)
     else:
         txt = ''
-    args = "bash dm.sh {} tgstation.dme".format(txt)
+    args = "bash tools/travis/dm.sh {} tgstation.dme".format(txt)
```
Comment (lordpidey) [inline tools/linux_build.py]: Waitwhat?  Why's this here.

Comment (JJRcop) [inline tools/linux_build.py]: atomic prs pls

Comment (coiax) [inline tools/linux_build.py]: I need to change my buildscript so it works again. So every PR I make, I include the change until one of them are merged. So this is already here.

**Comment by Iamgoofball** [comment]
:-1:

stop encouraging friendly wizard

File: code/game/gamemodes/antag_spawner.dm [hunk 2]
```diff
@@ -228,6 +228,13 @@
 	icon = 'icons/obj/wizard.dmi'
 	icon_state = "vial"
 
+	var/shatter_msg = "<span class='notice'>You shatter the bottle, no \
+		turning back now!</span>"
+	var/veil_msg = "<span class='warning'>You sense a dark presence lurking \
+		just beyond the veil...</span>"
```
Comment (PKPenguin321) [inline code/game/gamemodes/antag_spawner.dm]: just curious as to why you even bother splitting these into two lines. they're not even that long

**Pull request merged**